fcrepo-kernel-modeshape/src/test/java/org/fcrepo/kernel/modeshape/spring/DefaultPropertiesLoaderTest.java

```diff
@@ -132,7 +132,7 @@ public void testNoDefault() {
 
     @Test
     public void testValueSetForNoDefault() {
-        final String value = "/absolute/path";
+        final String value = new File("/absolute/path").getAbsolutePath();
```

Comment:
Build failed at this. Expected:<[]/absolute/path> but was:<[C:\fcrepo4\fcrepo-kernel-modeshape\fcrepo4-data\]/absolute/path>

Reply:
That is fascinating - what version of WIndows are you on?  I'm on Windows 10, and it worked fine for me.

In any case, I pushed an update that takes a slightly different approach.  Please see if it works for you.

Reply:
I'm on Windows 10, surface book.